content/ref/python/data-types/_index.md

```diff
@@ -14,7 +14,7 @@ flexible containers for information, like tables and HTML, and more.
 For more on logging media, see [our guide](https://docs.wandb.com/guides/track/log/media)
 
 For more on logging structured data for interactive dataset and model analysis,
-see [our guide to W&B Tables](https://docs.wandb.com/guides/tables/).
```

Comment:
Just a heads up that anything in content/ref/python/ is autogenerated from [wandb/wandb/sdk](https://github.com/wandb/wandb/tree/main/wandb/sdk), so it'll get rewritten on the next SDK Release.  To preserve changes, we'll need to update their respective docstrings.

Reply:
https://github.com/wandb/wandb/pull/9502